lib/commands/general.js

```diff
@@ -234,6 +235,10 @@ commands.mobileViewportScreenshot = async function () {
   return await this.getViewportScreenshot();
 };
 
+commands.mobileViewPortRect = async function () {
```

Comment:
Please add a name to the function - this way it's easier to debug such calls

Reply:
What do you mean? How do I "add a name to the function"? It's name is `mobileViewPortRect`, isn't it?